src/emulator/gui/src/profiles_manager_dialog.cpp

```diff
@@ -0,0 +1,189 @@
+// Vita3K emulator project
+// Copyright (C) 2019 Vita3K team
+//
+// This program is free software; you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation; either version 2 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License along
+// with this program; if not, write to the Free Software Foundation, Inc.,
+// 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+#include "private.h"
+
+#include <config/functions.h>
+#include <host/state.h>
+#include <misc/cpp/imgui_stdlib.h>
+#include <util/log.h>
+
+namespace gui {
+
+void draw_profiles_manager_dialog(GuiState &gui, HostState &host) {
+    ImGui::PushStyleColor(ImGuiCol_Text, GUI_COLOR_TEXT_MENUBAR_OPTIONS);
+    ImGui::Begin("Profiles Manager", &gui.configuration_menu.profiles_manager_dialog, ImGuiWindowFlags_AlwaysAutoResize);
+    ImGui::PopStyleColor();
+    auto update_online_id = std::find(host.cfg.online_id.begin(), host.cfg.online_id.end(), gui.online_id);
+
+    // Online ID
+    ImGui::InputTextWithHint("##online_id", "Write your online ID", &gui.online_id);
+    if (ImGui::IsItemHovered())
+        ImGui::SetTooltip("Set your Online ID with maximum 16 character.");
+    ImGui::SameLine();
+    if (ImGui::Button("Add Online ID") && !gui.online_id.empty())
+        ImGui::OpenPopup("Online ID");
+    ImGui::PushStyleColor(ImGuiCol_Text, GUI_COLOR_TEXT_MENUBAR_OPTIONS);
+    if (ImGui::BeginPopupModal("Online ID", NULL, ImGuiWindowFlags_AlwaysAutoResize)) {
+        ImGui::PopStyleColor();
+        if (gui.online_id.length() > 16)
+            ImGui::Text("Error: Need reduce longer for online ID.\nActuall longer: %d.", gui.online_id.length());
```

Comment:
"Longeur" in English is "Length". It should be "Need to reduce..." Actual has one "L."